frontend/package.json

```diff
@@ -61,7 +59,6 @@
   "devDependencies": {
     "@testing-library/dom": "^7.2.0",
     "@testing-library/react": "^9.5.0",
-    "cypress": "^4.3.0",
```

Comment:
I would propose to leave cypress and the one test in there. I believe it would encourage someone who want to start adding tests for the project. Does it affect the project in any way if we leave it there ?